steward/src/commands/reinvest.rs

```diff
@@ -0,0 +1,66 @@
+use std::{convert::TryFrom, path, sync::Arc, time::Duration};
+
+use abscissa_core::{clap::Parser, Application, Command, Runnable};
+
+use ethers::prelude::*;
+
+use signatory::FsKeyStore;
+
+use crate::{allocation, cellars::uniswapv3::UniswapV3CellarState, gas::CellarGas, prelude::*};
+
+/// Cellars reinvest command
+#[derive(Command, Debug, Parser)]
+pub struct ReinvestCommand {
```

Comment:
Delete this file.

Reply:
I deleted these I dont know what happened. here we go again